src/explorer/CosmosDBTreeItem.ts

```diff
@@ -77,7 +88,11 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
     }
 
     public async createChildImpl(showCreatingTreeItem: (label: string) => void): Promise<AzureTreeItem<ISiteTreeRoot>> {
-        const databaseToAdd = await ext.cosmosAPI.pickDatabase();
+        const options: PickTreeItemOptions & { resourceType: 'Database' } = {
```

Comment:
nit: You should be able to just do: `const options: PickTreeItemOptions = {`

Reply:
Compile error I get:
![image](https://user-images.githubusercontent.com/15349723/48295468-d22cee80-e440-11e8-92a0-ee648bcb1e03.png)